ui/lib/pki/addon/templates/roles/index.hbs

```diff
@@ -6,8 +16,8 @@
   </ToolbarActions>
 </Toolbar>
 
-{{#if (gt this.model.length 0)}}
-  {{#each this.model as |pkiRole|}}
+{{#if (gt this.model.model.length 0)}}
```

Comment:
I suspect there's a better way to do this, but I couldn't figure out how shape the model without giving it a label. I tried spread attrs, and that didn't work.

Reply:
You can change the key specified in `RSVP.hash` to something other than model